File: snippets/contact-center/create-enqueue_call-endpoint/create-enqueue_call-endpoint.php
<?php
header('Content-type: application/xml');

require 'vendor/autoload.php';
use Twilio\Twiml;

$department = [
1 => "sales",
2 => "support",
3 => "marketing"
];

$workflow_array = [
1 => getenv("TWILIO_ACME_SALES_WORKFLOW_SID"),
2 => getenv("TWILIO_ACME_SUPPORT_WORKFLOW_SID"),
3 => getenv("TWILIO_ACME_MARKETING_WORKFLOW_SID")
];

$response = new  Twiml();

$choice = $_REQUEST['Digits'];

$enqueue = $response->enqueue([workflowSid=>$workflow_array[$choice]]);
	$enqueue->task('{"selected_product":"'.$department[$choice].'"}');

echo $response;

File: snippets/contact-center/create-enqueue_call-endpoint/create-enqueue_call-endpoint.java
public void service(HttpServletRequest request, HttpServletResponse response) throws IOException {
    
    Map<String, String> departments = new HashMap<String, String>();
    departments.put("1", "support");
     departments.put("2", "sales");
      departments.put("3", "marketing");
    

    Task task = new Task.Builder().data("{\"selected_product\":\""map.get(request.getParameter("Digits"))"\"}").build();

    Enqueue enqueue =
        new Enqueue.Builder()
        .task(task).workflowSid("WWXXXXXXXXXXXXXXXXXXXXXXXXX").build();

    VoiceResponse twiml = new VoiceResponse.Builder().enqueue(enqueue).build();

    response.setContentType("application/xml");
    try {
      response.getWriter().print(twiml.toXml());
    } catch (TwiMLException e) {
      e.printStackTrace();
    }